test/testlib/minikube_utilities.go

```diff
@@ -923,17 +952,19 @@ func VerifyAdminKvSetAndGet(t *testing.T, podName string, namespaceName string)
 	)
 	require.NoError(t, err, "Could not set KV value")
 	elapsed := time.Since(start)
-
-	require.True(t, elapsed.Seconds() < 2.0, fmt.Sprintf("KV set took longer than 2s: %s", elapsed))
+	if elapsed.Seconds() > 2.0 {
```

Comment:
@NikTJ777 wanted to have these as regular tests to verify that RAFT is not unexpectedly slow. Nobody looks at the log output.

Reply:
It isn't an error for a Raft command to take longer than 2 seconds, and we don't really have any control over what kind of hardware the tests run on in Travis CI. I think catching performance regressions is outside the scope of a continuous test suite.

Reply:
Actually, the KV timeout test was _not_ to test for performance regressions, but to bail out of the test with a useful explanation in the event that the hardware that Travis has allocated to us is just too slow.

The goal is to avoid chasing our tails trying to diagnose (false) failures in tests that are actually caused by the hardware being so slow that the admin layer and KV store are causing otherwise correct tests to timeout and fail.